scripts/build_python.sh

```diff
@@ -30,6 +34,9 @@ echo_bold_white() {
     echo -e "\033[1;37m$*\033[0m"
 }
 
+# Assume MaxOS BigSur is used if compiling on MAC
```

Comment:
Also, that's not a great assumption....  Why do we need to assume that?

Reply:
I don't understand this either.

Reply:
https://github.com/project-chip/connectedhomeip/pull/5202

Reply:
My logic was that assuming newest OS is more common than an older one since we already had docs about things not compiling and I hit this myself.

I can drop this.